dev-release

```diff
@@ -70,7 +70,7 @@ const args = process.argv.slice(2);
 
   console.log("5. Build design & login SDKs");
   await run("yarn workspace @gooddollar/good-design install");
-  await run("yarn workspace @gooddollar/good-design build");
+  await run("yarn workspace @gooddollar/good-design build:release");
```

Comment:
why not build:release other packages? pls explain this change

Reply:
It was first an attempt to see if the difference in release was causing the storybook deployment failures on feature-branches.

I left it there because it just releases a more clean package.

There is no such thing implemented for good-login (and if I release I always use --skip-login as that package is not really up to date with anything)

Reply:
ok